SYCL/SpecConstants/2020/non_native/cuda.cpp

```diff
@@ -1,6 +1,6 @@
 // REQUIRES: cuda
 
-// RUN: %clangxx -fsycl -fsycl-targets=nvptx64-unknown-unknown-sycldevice %S/Inputs/common.cpp -o %t.out
+// RUN: %clangxx -fsycl -fsycl-targets=nvptx64-nvidia-cuda %S/Inputs/common.cpp -o %t.out
 // RUN: env SYCL_DEVICE_FILTER=cuda %t.out
 
 // TODO: enable this test then compile-time error in sycl-post-link is fixed
```

Comment:
I noticed that target triple is wrong. Can it be the reason of "compile-time error in sycl-post-link"?
Should I try to enable this test?

Reply:
I can't evaluate if `sycl-post-link` would be the tool affected by the incorrect triple or not, but personally, wouldn't object against a CI check as to that.